lib/backend/k8s/resources/ippool_test.go

```diff
@@ -134,14 +123,4 @@ var _ = Describe("IP Pool conversion methods", func() {
 		Expect(kvp.Value).To(BeAssignableToTypeOf(&model.IPPool{}))
 		Expect(kvp.Value).To(Equal(kvp1.Value))
 	})
-
```

Comment:
same as above, we added CIDR to the Spec so we don't need the conversion this test was testing